packages/client/hmi-client/src/components/dataset/tera-dataset-datatable.vue

```diff
@@ -3,7 +3,7 @@
 		<!-- Toggle histograms & column summary charts -->
 		<div class="datatable-toolbar">
 			<span class="datatable-toolbar-item">
-				{{ rawContent.headers.length || 'No' }} columns | {{ rawContent.csv.length || 'No' }} rows
+				{{ rawContent.headers.length || 'No' }} columns | {{ rawContent.csv.length - 1 || 'No' }} rows
```

Comment:
This doesn't seem right? Why do we have `length - 1`?

Reply:
Because the first row are the headers?

Reply:
Looks like the backend include headers in csv.